src/panels/config/logs/error-log-card.ts

```diff
@@ -29,6 +30,9 @@ class ErrorLogCard extends LitElement {
               <mwc-button raised @click=${this._refreshErrorLog}>
                 ${this.hass.localize("ui.panel.config.logs.load_full_log")}
               </mwc-button>
+              <mwc-button raised @click=${this._downloadLogs}>
```

Comment:
```suggestion
              <mwc-button @click=${this._downloadLogs}>
```

![image](https://user-images.githubusercontent.com/2270505/151131202-20301cb9-9c23-412f-a38d-04ed9fc0b880.png)